### ISSUE TITLE
Allow user to specify a profiles.yml

### ISSUE THREAD

**Issue by owlas** [lightdash/lightdash#198]
To source projects from dbt cloud scheduler api, we'll need to construct or own `profiles.yml` in order to run queries.

Users may also want to specify a different `profiles.yml` for lightdash with fewer privileges than the profile they use for their ELT pipeline (i.e. not have write access).

**Comment by TuringLovesDeathMetal** [comment]
@owlas - I feel like we've done this work this week/last week...but I can't find an issue to map it to

**Comment by TuringLovesDeathMetal** [comment]
@owlas is this issue still a thing?

**Comment by ZeRego** [comment]
@TuringLovesDeathMetal I don't think so. This ticket was before we generated our own profiles.yml. Closing it
